voltdbclient/url.go

```diff
@@ -25,9 +26,23 @@ func parseURL(src string) (*url.URL, error) {
 			}
 		}
 	} else {
-		if u.Port() == "" {
+		if getPort(u.Host) == "" {
 			u.Host = u.Host + ":21212"
 		}
 	}
 	return u, nil
 }
+
+func getPort(host string) string{
```

Comment:
@gernest @nshi 👍 Yeah this is the logic that go uses in url library.

>It's not necessary to copy the implementation here.
Normally, url.Port() was used here, but then i've learnt that u'r not supporting >=1.8 version. & then i did this change.
Once you support >=1.8 version of go, we can use Port() method.